Assets/Scripts/Infrastructure/NetworkGuid.cs

```diff
@@ -3,16 +3,10 @@
 
```

Comment:
adding note here, let's not forget the checklist for PRs, I'd expect a changelog for this